web_tests/pages/dashboard_page.py

```diff
@@ -0,0 +1,22 @@
+from selenium.webdriver.common.by import By
+from web_tests.pages.base_page import BasePage
+
+
+class DashboardPage(BasePage):
+    # URL = 'https://opensource-demo.orangehrmlive.com/web/index.php/dashboard'
+
+    def __init__(self, driver):
+        super().__init__(driver)
+
+    DASHBOARD_TITLE = (By.XPATH, '//h6[text()="Dashboard"]')
```

Comment:
Using text in locators is not good idea. To get title you could use just //h6 and verify that correct title is set in test. And for checking page is displayed, nice to find some unique element.
If using text in real project need to add functionality so text in framework will be changed according set locale.